core/src/main/java/brooklyn/util/crypto/FluentKeySigner.java

```diff
@@ -42,7 +44,9 @@
  * X509v3CertificateBuilder drags in an add'l dependency (bcmail) and is harder to use. */
 @SuppressWarnings("deprecation")
```

Comment:
i'll refactor to use the fully qualified class name everywhere, that way we can suppress deprecations on a per-line basis not class-wide